install/kots/Makefile

```diff
@@ -16,7 +16,7 @@ create_dev_release:
 		exit 1; \
 	fi
 
-	replicated release create --lint --ensure-channel --yaml-dir ${YAML_DIR} --promote ${REPLICATED_DEV_CHANNEL}
+	replicated release create --lint --ensure-channel --yaml-dir ${YAML_DIR} --promote ${REPLICATED_DEV_CHANNEL} --version="$(shell date +%s)"
```

Comment:
Is this temporary or meant to be merged?

Reply:
To be merged. This is only for when a developer wants to push a release from their dev branch (eg, `dev-sje`) - airgapped releases need unique version numbers, so this is what this is for.

It won't be used for releases to the stable/beta/unstable channels